server/src/hardware/decoder/decoderfunction.cpp

```diff
@@ -125,6 +127,7 @@ void DecoderFunction::typeChanged()
   const bool editable = contains(m_decoder.world().state.value(), WorldState::Edit);
   bool momentaryOrHold = (type == DecoderFunctionType::Momentary || type == DecoderFunctionType::Hold);
   Attributes::setEnabled(timeoutSeconds, editable && momentaryOrHold);
+  Attributes::setVisible(timeoutSeconds, editable && momentaryOrHold);
```

Comment:
Enabled should only be affected by `editable`
Visible should only be affected by `momentaryOrHold`

Then it behaves the same like other properties. (I should document that some where...)

Reply:
Done.